_posts/2019-11-20-tooling.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Setting up a local copy of the book"
+---
+
+In the spirit of [open science](https://www.fosteropenscience.eu/content/what-open-science-introduction), we are not only writing this book in the open, but also hoping to tap into
+[user-led innovation](https://innovationmanagement.se/2011/02/21/eric-von-hippel-on-innovation/). Towards that end, we want to give interested users the ability to setup the tooling
+to run the book locally on their own machines.
+
+We have developed a docker container that has all the software packages and instructions for building the book. Rather than having to install all these packages independently on your operating system, the container bundles all of these for us. Docker will enable you to run this container inside your host operating system.
+
+Below we outline the instructions for setting up your machine and running the container. 
+
+
+## Assumptions
+
+ 1. You have admin rights over your machine
+ 2. You are running either Windows 10 Pro, macOS, or Linux
```

Comment:
Should we give an option for those who don't meet these requirements? Arguably, that'll be quite a few of our readers.